webpack/scenes/ChangeContentSource/components/cs_form.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
```

Comment:
A lot of these files are using snake_case file names which is not consistent with the rest of the react app. take a look at some other files for guidance there